src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActivityFeed-test.js

```diff
@@ -2,10 +2,12 @@ import * as React from 'react';
 import { shallow } from 'enzyme';
 
 import ActivityFeed from '../ActivityFeed';
+import { scrollIntoView } from '../../../../../utils/dom';
 
 jest.mock('../../Avatar', () => 'Avatar');
 jest.mock('../ActiveState', () => 'ActiveState');
 jest.mock('lodash/uniqueId', () => () => 'uniqueId');
```

Comment:
I realize this is existing code, but can we mock in the same order we import, with third-party libraries first?